Makefile

```diff
@@ -0,0 +1,42 @@
+JSONNET_FILES=$(shell find . -type f -not -path './vendor/*' \( -name '*.libsonnet' -o -name '*.jsonnet' \))
+
+.DEFAULT_GOAL := all
+
+.PHONY: check
+check:
+	@echo Checking...
+	@echo $(JSONNET_FILES) | xargs -L1 jsonnetfmt -i
+	@git diff -s --exit-code . || (echo "Build failed: one of more files aren't staged to be committed." && git diff && exit 1)
```

Comment:
As commented in an earlier PR, can we not suppress these diffs, I'd like to see what's going on.

Reply:
It is doing that, but only if it fails. Look at the `&& git diff && exit 1` at the end of the command.